demosymfonyform/views/templates/admin/form.html.twig

```diff
@@ -0,0 +1,51 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% form_theme demoConfigurationForm 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit.html.twig' %}
+
+{% block content %}
+  {{ form_start(demoConfigurationForm) }}
+  <div class="card">
+    <div class="card-block row">
+      <div class="card-text">
+          {{ form_widget(demoConfigurationForm) }}
+      </div>
+    </div>
+    <div class="card-footer">
+      <div class="d-flex justify-content-end">
+        <button class="btn btn-primary float-right" id="save-button">
+          {{ 'Save'|trans({}, 'Admin.Actions') }}
+        </button>
+      </div>
+    </div>
+  </div>
+  {{ form_end(demoConfigurationForm) }}
+{% endblock %}
+
+{% block javascripts %}
+  {{ parent() }}
+  <script src="{{ asset('../modules/demosymfonyform/views/js/form.js') }}"></script>
```

Comment:
oh `asset` works for modules ? 😮  I tried to do it once, I could not

awesome !

Reply:
Yeah it's been done in `ps_linklist`, I wonder if we could improve this, the relative path is not very reliable imho 😅
I wonder if it would be possible to have some sort of alias like for twig paths `@Modules/...` that would be even nicer

Reply:
As far as I see assets doesn't use any "variables" for pathing anywhere else in PrestaShop. In this case do we need to modify the way assets builds path to make "@Modules" work?

Reply:
would be great to be consistent with front-office, for example: `module:name/path/something.js`